src/sign.ts

```diff
@@ -56,7 +57,7 @@ export class Signer {
     // Calculate artifact digest
     const digest = crypto.hash(payload);
 
-    const certificateB64 = enc.base64Encode(certificate);
+    const certificateB64 = enc.base64Encode(certificate[0]);
```

Comment:
nit could we make it more explicit we are fetching the leaf here? e.g.

```suggestion
    const = leafCertificate = certificate[0]
    const certificateB64 = enc.base64Encode(leafCertificate);
```